Xamarin.Forms.Platform.iOS/VisualElementPackager.cs

```diff
@@ -141,34 +143,59 @@ protected virtual void OnChildRemoved(VisualElement view)
 			viewRenderer.Dispose();
 		}
 
-		void EnsureChildrenOrder()
+		void EnsureChildrenOrder(VisualElement element = null)
```

Comment:
Could we add some tests validating the logic to order and ensure the order of the elements?